runtime/vm/createramclass.cpp

```diff
@@ -1586,14 +1599,14 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 			}
 
 			/* ensure that the superclass isn't an interface or final */
-			if ((superclass->romClass->modifiers & J9AccFinal) != 0) {
+			if (J9_ARE_ANY_BITS_SET(superclass->romClass->modifiers, J9AccFinal)) {
 				Trc_VM_CreateRAMClassFromROMClass_superclassIsFinal(vmThread, superclass);
-				setCurrentExceptionForBadClass(vmThread, superclassName, J9VMCONSTANTPOOL_JAVALANGVERIFYERROR);
+				setCurrentExceptionForBadClass(vmThread, superclassName, J9VMCONSTANTPOOL_JAVALANGVERIFYERROR, "The final superclass can't be inherited");
```

Comment:
These should be NLS messages

Reply:
"The final superclass can't be inherited" -> "Final superclass can't be extended"

Reply:
Previously I was considering whether it is necessary to go with NLS messages (withdrew the changes due to too many changes introduced) but I agreed to modify in this case as it makes more sense and clear to us. Already updated the whole solution with the suggested pattern below.

Reply:
The output from the new changes as follows:
```
jdk_openj9/bin/java -cp . X
Error: LinkageError occurred while loading main class X
        java.lang.VerifyError: Final superclass Y can't be extended
```